### PR TITLE
Submission of ecospat

### PR DESCRIPTION
**Pull request by ecospat**
bias: ecospat.occ.desaggregation to thin data
backg_sample: ecospat.sample.envar to add env values to a species dataframe
data_partitioning: ecospat.ESM.EnsembleEvaluation, ecospat.poolingEvaluation, ecospat.CommunityEval
mod_fit: ecospat.ESM.Modeling
mod_ensemble: ecospat.ESM.EnsembleModeling, ecospat.CCV.modeling
mod_evaluate: same as data_partitioning
pred_general: ecospat.ESM.Projection
pred_extrapolation: ecospat.climan
pred_inspect: ecospat.ESM.VarContrib, ecospat.ESM.responsePlot
post_processing: ecospat.grid.clim.dyn

### CODE REVIEW

**Comment by sylvainschmitt** [comment]
@ecospat , thanks for your submission. Please fix the following:

> -For package ecospat, title is defined as 'ecospat: Spatial Ecology Miscellaneous Methods' but should be defined as 'Spatial Ecology Miscellaneous Methods' according to CRAN.
-For package ecospat, maintainer is defined as 'Olivier Broennimann <olivier.broennimann at unil.ch>' but should be defined as 'Olivier Broennimann <olivier.broennimann@unil.ch>' according to CRAN.
-For package ecospat, description is defined as 'Collection of R functions and data sets for the support of spatial ecology analyses with a focus on pre, core and post modelling analyses of species distribution, niche quantification and community assembly. Written by current and former members and collaborators of the ecospat group of Antoine Guisan, Department of Ecology and Evolution (DEE) and Institute of Earth Surface Dynamics (IDYST), University of Lausanne, Switzerland.' but should be defined as 'Collection of R functions and data sets for the support of spatial ecology analyses with a focus on pre, core

I'm looking for reviewers: @EduardoArle , @gepinillab , @hannahlowens , @sgvignali, @codelab3 , @adamlilith, @MayaGueguen, @danlwarren , @AMBarbosa , @AndreaSanchezTapia, @danlwarren  are you available? Please have a look at the [reviewing guidelines](https://github.com/sylvainschmitt/sdmverse/blob/main/REVIEWING.md). Do not hesitate to ask further informations to @ecospat if the justification is a bit too short.

File: inst/extdata/packages/ecospat.yaml
Comment (jamiemkass) [inline inst/extdata/packages/ecospat.yaml:20]: The function `ecospat.sample.envar` gets env raster values for occurrence points, but does not sample background values. However, the function `ecospat.rand.pseudoabsences` does, so this can stay "yes".

File: inst/extdata/packages/ecospat.yaml
Comment (jamiemkass) [inline inst/extdata/packages/ecospat.yaml]: The functions listed don't seem to do data partitioning, but pool the results of CV for evaluation. Data partitioning would be assigning folds to data. So I think this should be "no".

File: inst/extdata/packages/ecospat.yaml
Comment (jamiemkass) [inline inst/extdata/packages/ecospat.yaml]: Wouldn't `ecospat.SESAM.prr` qualify for a "yes" here?

**Review by jamiemkass — CHANGES_REQUESTED** [review]
Two suggestions for changes.

**Comment by sylvainschmitt** [comment]
Your contribution can be integrated without further edits. After merging your pull request, you will be added to the repository contributors. Please expect to be contacted for a review request in the future.

**Pull request merged**
